test/util/ndarray.hpp

```diff
@@ -23,7 +23,6 @@
 #pragma once
```

Comment:
@diyessi You might want to check this file to make sure what I've done makes sense.